### ISSUE TITLE
[Feature request]:MD.Emu CPU Overclocking

### ISSUE THREAD

**Issue by TheBoxGame** [Rakashazi/emu-ex-plus-alpha#1401]
### Describe the feature

@Rakashazi could you add cpu overclocking to make games run faster, for example in road rash 2 the game slows down and with overclocking it runs smoother

**Comment by Dim7x0** [comment]
! Very important feature and 
very necessary!

**Comment by TheBoxGame** [comment]
@Dim7x0 yes I just don't know why the rakshazi don't want to give it back and it would be useful
